### PR TITLE
fix(context-menu) Hide toolbars when participant context menu opened

### PR DESCRIPTION
**Pull request by horymury**
- hide toolbars only when in tile view
- fix community issue: https://github.com/jitsi/jitsi-meet/issues/9818

### CODE REVIEW

File: react/features/video-menu/components/web/LocalVideoMenuTriggerButton.js [hunk 1]
```diff
@@ -81,6 +88,11 @@ class LocalVideoMenuTriggerButton extends Component<Props> {
      */
     popoverRef: Object;
 
+    /**
+     * Preserve the intial toolbox state.
+     */
+    toolboxEnabled: boolean;
```
Comment (vp8x8) [inline react/features/video-menu/components/web/LocalVideoMenuTriggerButton.js:94]: Could you please rename this property to something like `initialToolboxState`?  It's a bit confusing to have `toolboxEnabled` and `_props.toolboxEnabled`.

File: react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.js [hunk 2]
```diff
@@ -131,6 +138,11 @@ class RemoteVideoMenuTriggerButton extends Component<Props> {
      */
     popoverRef: Object;
 
+    /**
+     * Preserve the intial toolbox state.
+     */
+    toolboxEnabled: boolean;
```
Comment (vp8x8) [inline react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.js:144]: ditto

**Comment by vp8x8** [comment]
Left a minor thing. Other than that it looks good.

**Review by vp8x8 — APPROVED** [review]

**Comment by horymury** [comment]
Jenkins please test this please

**Comment by ashiqhassan95** [comment]
I think this fixes the issue https://github.com/jitsi/jitsi-meet/issues/9752

Please correct me if not.

**Comment by horymury** [comment]
> @horymury
> 
> I think this fixes the issue #9752
> 
> Please correct me if not.
@ashiqhassan95 yes but not 100%, when context menu is opened, the conference info and the toolbar panes will stay hidden so the menu items will be accessible. But you'll need to be able to hover over the 3 dots menu 'under' the conference info header initially.

**Pull request merged**